app/views/search/create.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Search Results</h1>
+<p>Loading...</p>
+
+<ul>
+<% @results.each do |result| %>
+  <li> <h2> <%= result.title %> </h2>
+     <%= result.body %>
```

Comment:
It might be better to have a link to the result so we can view the item in their respective show pages.